docs/package.json

```
@@ -5,7 +5,8 @@
   "scripts": {
     "clean": "gatsby clean",
     "develop": "gatsby develop",
-    "build": "pushd .. && npm install && popd && cross-env NODE_ENV=production gatsby build --prefix-paths"
+    "build": "pushd .. && npm install && popd && cross-env NODE_ENV=production gatsby build --prefix-paths && npm run add:storybook",
+    "add:storybook": "cd .. && ./script/deploy-storybook"
```

Should we call this `build` instead of `add`/`deploy` since that's closer to what's happening in these scripts?
```suggestion
    "build:storybook": "cd .. && ./script/build-storybook"
```

---

Yeah, I wasn't sure if i should call it build, because it's so focused on building it inside the docs and doesn't work standalone.

maybe we should call it `publish-storybook`?

---

Ah, I understand your hesitation but `deploy` and `publish` both make me think the script might actually deploy or publish the site (e.g. make some calls to Vercel). 

`build` seems fine to me since we only ever need to build storybook inside our docs. Also having this script inside the `docs/package.json` implies that the build is specifically for docs.

---

Makes sense! Changed to build and shipped :shipit: